gpii/node_modules/userListeners/README.md

```diff
@@ -0,0 +1,34 @@
+# User listeners
+
+The user listeners start when GPII does.
+
+## RFID Listener
+
+`gpii.userlisteners.pcsc`
+
+* [pcsc.js](src/pcsc.js)
+* Uses [@pokusew/pcsclite](https://www.npmjs.com/package/@pokusew/pcsclite) to access the card reader.
+  This has build dependencies on Linux, which prevents it being tested easily in CI, so the actual reference to this package is in gpii-windows.
+* Cards supported (tested):
+  * Mifare Classic 1k and 4k
+  * NTAG203/213/216
+  * Mifare Ultralight.
+
+### Testing
+
+* Dry testing (with an emulated card) in [pcscTests.js](test/pcscTests.js)
+* There are tests that use a real card in [cardTests/](test/cardTests). These are not tested in CI (they require a
+reader), and only useful for developing/debugging.
```

Comment:
I observed how the manual tests run, can you add a little description about how this manual testing can be done?